vkrunner/vr-test.c

```diff
@@ -1039,7 +1039,7 @@ set_buffer_subdata(struct test_data *data,
         vr_flush_memory(data->window->context,
                         buffer->memory_type_index,
                         buffer->memory,
-                        command->set_push_constant.offset,
+                        command->set_buffer_subdata.offset,
                         command->set_buffer_subdata.size);
```

Comment:
This change fixes validation problems that I'm seeing with the offset here, but I still get problems with the size. Specifically, it needs to be a multiple of the non-coherent atom size, but it typically isn't in the test. I can add some code to fix that, but is it important that this size be used here, or could it just be changed to VK_WHOLE_SIZE?